docs/source/usage/new_user_guide.rst

```diff
@@ -4,7 +4,7 @@ New User Guide
 Prerequisites
 -------------
 
-The first step is to gain access to an ESI installation. After you do, the ESI operator will send you credentials and instructions regarding how to access your account.
+The first step is to gain access to an ESI installation. After you do, the ESI operator will send you credentials and instructions regarding how to access your account. If you are interested in MOC ESI installation, read the `MOC-ESI`_ section for further information.
```

Comment:
Ah, I think I might have said things backwards - I don't think the new user guide should point at the MOC ESI section; I think the MOC ESI section should point at the new user guide (perhaps right after you mention the Horizon dashboard, since you can't actually do much with ESI in Horizon).

Reply:
Fixed.